sdk/eventhub/azure-eventhub/azure/eventhub/_transport/_uamqp_transport.py

```diff
@@ -107,24 +108,27 @@ class UamqpTransport(AmqpTransport):    # pylint: disable=too-many-public-method
         def build_message(**kwargs):
             """
             Creates a uamqp.Message with given arguments.
-            :rtype: uamqp.Message
+            :rtype: ~uamqp.Message
```

Comment:
sorry this one is def on me since I typed the docstring as `~uamqp` here. I think it should be `uamqp.Message` etc. here and everywhere in this file? I'm not super clear on when the `~` should be used 🤔